app/src/components/common/NetworkSelector.vue

```diff
@@ -4,7 +4,9 @@
       v-for="network in mainnets"
       :key="network.chain_id"
       class="network-item"
-      :class="{ selected: networkId === network.id }"
+      :class="{
+        selected: networkId === network.id && $route.name !== `notifications`,
```

Comment:
same. can we add `networkIndependent` to the route?

Reply:
what do you mean?

Reply:
Re https://github.com/luniehq/lunie/pull/4550#discussion_r457580363